client/components/Login.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
```

Comment:
Redux changes are not required here. Please remove these changes. They are not part of your ticket.